documentation/modules/auxiliary/scanner/http/scraper.md

```diff
@@ -0,0 +1,26 @@
+## Description 
+This module scrapes data from a specific web page based on a regular expression.
+
+## Verification Steps
+
+1. Do: ```use auxiliary/scanner/http/scraper```
+2. Do: ```set RHOSTS [IP]```
+3. Do: ```run```
+
```

Comment:
This is missing a `## Options` section, where you'd mention the PATTERN and call out the default instead of putting it in the scenarios section.